assignments/callbacks.js

```diff
@@ -20,25 +20,35 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
   });
 
 */
-
-
+// function cb(arg){
+//   console.log(arg)
+// }
 function getLength(arr, cb) {
+  return cb(arr.length)
   // getLength passes the length of the array into the callback.
+  
 }
+getLength (items, cb)
```

Comment:
You would want to replace `cb` with some function that You want to call as callback, for example `console.log`.